manager/src/main/java/io/aklivity/zilla/manager/internal/commands/install/ZpmInstall.java

```diff
@@ -212,26 +219,21 @@ public void invoke()
             logger.error(String.format("Error: %s", ex.getMessage()));
             throw new RuntimeException(ex);
         }
-        finally
-        {
-            if (!silent)
-            {
-                logger.sumupProblems();
-            }
-        }
     }
 
-    private void readSettings(
-        Path settingsDir) throws IOException, PlexusCipherException
+    private List<RemoteRepository> getRemoteRepositories(
+        Path settingsDir,
+        List<ZpmRepository> repositories) throws IOException, PlexusCipherException
```

Comment:
The logic in this method will change with the switch to use standard maven credentials instead.